### ISSUE TITLE
The exception was a java.lang.NoClassDefFoundError : tlc2/tool/EvaluatingValueTest

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlaplus#964]
Running `dist/tla2tools.jar` on any specification in the `test-model` directory encounters the following error:

```java
Starting... (2024-06-20 17:04:37)
Error: TLC threw an unexpected exception.
This was probably caused by an error in the spec or model.
See the User Output or TLC Console for clues to what happened.
The exception was a java.lang.NoClassDefFoundError
: tlc2/tool/EvaluatingValueTest
```

This error occurs because `dist/tla2tools.jar `does not include the class files compiled into `test-classes` including `EvaluatingValueTest` (this is the distribution!).  When attempting to run any spec in `test-model`, TLC's dynamic module loading is triggered, which fails because the required class is missing.

To work around this issue, copy the spec out of the test-model directory. Running specs in any other directory works fine, provided that directory does not also trigger TLC's module loading of a missing class. 

This issue is noted for development information only and is not intended to be fixed.

**Comment by ahelwer** [comment]
Is the `test-model` directory name hardcoded in TLC or something?

**Comment by lemmy** [comment]
No, TLC automatically loads any Java module override it finds in the current working directory, which in this case is `test-model`.  Since there are tests for dynamic module loading, such as `EvaluatingValueTest`, TLC attempts to load these modules.
